tema1.js

```diff
@@ -0,0 +1,18 @@
+function removeProperty(obj, property) {
+
+    const newObj = {...obj};
+
+    delete newObj[property];
+
+    return newObj;
+}
+
+const obj = {a:1, b:2};
+const property = "b";
+
+const propertyRemoved = removeProperty(obj, property);
```

Comment:
```suggestion
const objModified = removeProperty(obj, property);
```
Ceea ce functia returneaza este tot un obiect. Nu e potrivit sa il numim "properyRemoved", ci mai degraba ceva ce incepe cu "object" pentru ca asta este de fapt.